### ISSUE TITLE
IntelliJ IDEA 2023.2  UnmarshalException: Error unmarshaling return header

### ISSUE THREAD

**Issue by RuslanSafa** [grisha9/gmaven-plugin#2]
An error occurs in the IDE.

IntelliJ IDEA 2023.2 (Ultimate Edition)
Build #IU-232.8660.185, built on July 26, 2023
Subscription is active until April 23, 2024.
Runtime version: 17.0.7+7-b1000.6 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 13.3.1
GC: G1 Young Generation, G1 Old Generation
Memory: 4024M
Cores: 12
Metal Rendering is ON
Registry:
    debugger.new.tool.window.layout=true
    ide.experimental.ui=true
    ide.balloon.shadow.size=0
    editor.minimap.enabled=true

Non-Bundled Plugins:
    com.mroche.JsonHelper (1.2.0)
    com.intellij.kubernetes (232.8660.158)
    ski.chrzanow.foldableprojectview (1.1.6)
    org.jetbrains.plugins.docker.gateway (232.8660.202)
    Key Promoter X (2023.2.0)
    org.jetbrains.settingsRepository (232.8660.88)
    com.robohorse.robopojogenerator (2.4.1)
    ru.rzn.gmyasoedov.gmaven (231.4)
    com.hardy.JsonTool (1.0.3)
    String Manipulation (9.10.0)
    GrepConsole (12.19.211.6693.0)
    com.chylex.intellij.inspectionlens (1.3.0)
    MavenRunHelper (4.23.222.2964.0)
    com.intellij.bigdatatools.core (232.8660.206)
    com.intellij.bigdatatools.kafka (232.8660.206)
    com.github.lppedd.idea-return-highlighter (0.8.3)
    izhangzhihao.rainbow.brackets (2023.3.3)
    cn.yiiguxing.plugin.translate (3.5.2)
    com.haulmont.jpab (2023.3.2-232)
    org.mapstruct.intellij (1.6.1)
    org.sonarlint.idea (8.5.1.75093)
    com.majera.intellij.codereview.bitbucket (2023.3.2.232)

Kotlin: 232-1.9.0-IJ8660.185

```java
java.rmi.UnmarshalException: Error unmarshaling return header; nested exception is: 
	java.io.EOFException
	at java.rmi/sun.rmi.transport.StreamRemoteCall.executeCall(StreamRemoteCall.java:255)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:165)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy29/jdk.proxy29.$Proxy464.getProjectModel(Unknown Source)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at com.intellij.execution.rmi.RemoteUtil.invokeRemote(RemoteUtil.java:146)
	at com.intellij.execution.rmi.RemoteUtil.access$400(RemoteUtil.java:22)
	at com.intellij.execution.rmi.RemoteUtil$1.lambda$invoke$0(RemoteUtil.java:130)
	at com.intellij.openapi.util.ClassLoaderUtil.computeWithClassLoader(ClassLoaderUtil.java:31)
	at com.intellij.execution.rmi.RemoteUtil.executeWithClassLoader(RemoteUtil.java:198)
	at com.intellij.execution.rmi.RemoteUtil$1.invoke(RemoteUtil.java:130)
	at jdk.proxy29/jdk.proxy29.$Proxy464.getProjectModel(Unknown Source)
	at ru.rzn.gmyasoedov.gmaven.server.GServerHelper.runMavenTaskInner(GServerHelper.kt:162)
	at ru.rzn.gmyasoedov.gmaven.server.GServerHelper.runMavenTask(GServerHelper.kt:149)
	at ru.rzn.gmyasoedov.gmaven.server.GServerHelper.runMavenTask$default(GServerHelper.kt:145)
	at ru.rzn.gmyasoedov.gmaven.server.GServerHelper.runTasks(GServerHelper.kt:73)
	at ru.rzn.gmyasoedov.gmaven.project.task.MavenTaskManager.executeTasks(MavenTaskManager.kt:40)
	at ru.rzn.gmyasoedov.gmaven.project.task.MavenTaskManager.executeTasks(MavenTaskManager.kt:17)
	at com.intellij.openapi.externalSystem.service.remote.RemoteExternalSystemTaskManagerImpl.lambda$executeTasks$0(RemoteExternalSystemTaskManagerImpl.java:43)
	at com.intellij.openapi.externalSystem.service.remote.AbstractRemoteExternalSystemService.execute(AbstractRemoteExternalSystemService.java:40)
	at com.intellij.openapi.externalSystem.service.remote.RemoteExternalSystemTaskManagerImpl.executeTasks(RemoteExternalSystemTaskManagerImpl.java:42)
	at com.intellij.openapi.externalSystem.service.remote.wrapper.ExternalSystemTaskManagerWrapper.executeTasks(ExternalSystemTaskManagerWrapper.java:35)
	at com.intellij.openapi.externalSystem.service.internal.ExternalSystemExecuteTaskTask.doExecute(ExternalSystemExecuteTaskTask.java:131)
	at com.intellij.openapi.externalSystem.service.internal.AbstractExternalSystemTask.execute(AbstractExternalSystemTask.java:149)
	at com.intellij.openapi.externalSystem.service.execution.ExternalSystemRunnableState.lambda$execute$1(ExternalSystemRunnableState.java:323)
	at com.intellij.openapi.application.impl.ApplicationImpl$2.run(ApplicationImpl.java:272)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.io.EOFException
	at java.base/java.io.DataInputStream.readByte(DataInputStream.java:273)
	at java.rmi/sun.rmi.transport.StreamRemoteCall.executeCall(StreamRemoteCall.java:241)
	... 38 more
`

**Comment by grisha9** [comment]
This is a consequence of https://github.com/grisha9/gmaven-plugin/issues/1

**Comment by grisha9** [comment]
Expected exception is successfully processed in GServerHelper#runMavenTaskInner, but the logging level "ERROR" results in an IDE error. lowered the logging level to WARN

**Comment by grisha9** [comment]
will fixed in 232.15 version
